src/algorithms/calorimetry/CalorimeterClusterMerger.cc

```diff
@@ -39,31 +39,32 @@ void CalorimeterClusterMerger::AlgorithmProcess() {
     auto& assoc2 = m_outputAssociations;
 
     if (!split.size()) {
-      if (false) {
-        LOG_INFO(default_cout_logger) << "Nothing to do for this event, returning..." << LOG_END;
+      if (m_logger->level() <=spdlog::level::debug) {
+        m_logger->debug("Nothing to do for this event, returning...");
+        //LOG_INFO(default_cout_logger) << "Nothing to do for this event, returning..." << LOG_END;
       }
       return;
     }
 
-    if (false) {
-      LOG_INFO(default_cout_logger) << "Step 0/1: Getting indexed list of clusters..." << LOG_END;
+    if (m_logger->level() <=spdlog::level::debug) {
+       m_logger->debug( "Step 0/1: Getting indexed list of clusters..." );
+      //LOG_INFO(default_cout_logger) << "Step 0/1: Getting indexed list of clusters..." << LOG_END;
     }
     // get an indexed map of all vectors of clusters, indexed by mcID
     auto clusterMap = indexedClusterLists(split, assoc);
 
     // loop over all position clusters and match with energy clusters
-    if (false) {
-      LOG_INFO(default_cout_logger) << "Step 1/1: Merging clusters where needed" << LOG_END;
+    if (m_logger->level() <=spdlog::level::debug) {
```

Comment:
`If` could be removed

Reply:
desire to leave as close to the original as possible, for now